deployments/examples/ocis_keycloak/docker-compose.yml

```diff
@@ -56,6 +56,7 @@ services:
     environment:
       # Keycloak IDP specific configuration
       PROXY_AUTOPROVISION_ACCOUNTS: "true"
+      GRAPH_LDAP_SERVER_WRITE_ENABLED: "false" # disable adding users via the oCIS user management
```

Comment:
Hm, I think this will fail. The auto provisioning feature in ocis uses the Graph API to create users. If `GRAPH_LDAP_SERVER_WRITE_ENABLED` is set to `false` the auto provisioning will also fail. We need to work out a different solution here.